data/migrations/013-recipients.js

```diff
@@ -9,7 +9,7 @@ exports.up = (knex) => {
     tbl.string('recipient_last_name').notNullable();
     tbl.date('recipient_date_of_birth').notNullable();
     tbl.boolean('recipient_veteran_status').notNullable();
-    tbl.boolean('qualify_status').notNullable();
+    tbl.boolean('active_status').notNullable();
```

Comment:
Thanks for making this change. "Qualify" was confusing because it does not refer to whether a client/recipient is qualified for a particular service. Instead I think the intent is to be able to hide particular records in lieu of deleting them since that may cause issues with displaying existing service log records if the underlying recipient record is fully deleted from the db.